sass/components/_timepicker.scss

```diff
@@ -24,13 +24,18 @@
 }
 
 .timepicker-text-container {
-	font-size: 4rem;
+  width: 200px;
+  font-size: 4rem;
 	font-weight: bold;
 	text-align: center;
 	color: rgba(255, 255, 255, 0.6);
   font-weight: 400;
 	position: relative;
   user-select: none;
+  input[type=text]{
+    color: rgba(255, 255, 255, 0.6);
+    direction: rtl;
+  }
```

Comment:
please fix indentation of the whole block

Reply:
Will be fixed

Reply:
Indentation of lines 28-30 is inconsistent.

Reply:
My editor shows it's aligned somehow, but I will fix accordingly.